src/main/java/io/pinecone/model/IndexMeta.java

```diff
@@ -0,0 +1,47 @@
+package io.pinecone.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class IndexMeta {
+    @JsonProperty("database")
+    private IndexMetaDatabase metaDatabase;
```

Comment:
**nit:** Maybe worth just calling this `database`? I think that's consistent with what we do in the node client as well. Just feels a little confusing to me at first glance. We're already in a class called `IndexMeta` so maybe just `database` is sufficient. We may not need the `@JsonProperty()` transform either.

Reply:
Thanks for the feedback, fixed!